src/components/sidebar/MyPools.tsx

```diff
@@ -138,23 +140,25 @@ const MyPools: React.FC = ({ ...restProps }) => {
             setActiveFilter(MyPoolsFilter.Invested)
           }}
         >
-          {`Invested (${getPools(userResponse, MyPoolsFilter.Invested).length})`}
+          {`Invested (${!isConnected ? 0 : getPools(userResponse, MyPoolsFilter.Invested).length})`}
```

Comment:
Is there a way to handle this within the getPools function? To avoid the ternary operator

Reply:
Yeah, great idea!